src/rabbit_fifo_client.erl

```diff
@@ -63,7 +65,7 @@
                 servers = [] :: [ra_server_id()],
                 leader :: maybe(ra_server_id()),
                 next_seq = 0 :: seq(),
-                last_applied :: maybe(seq()),
+                last_applied = -1 :: maybe_seq(),
```

Comment:
Should we document inline why it is the case?

Reply:
done